app/jsx/components/PublishedLocationComp.jsx

```diff
@@ -4,25 +4,47 @@ import React from 'react'
 
 class PublishedLocationComp extends React.Component {
   render() {
-    return (
-      <div className="c-publishedlocation">
+    let pub_loc_block = null
+    if (this.props.pub_web_loc.length > 0) {
+      pub_loc_block = 
         <div className="c-publishedlocation__location">
-          <a className="o-textlink__secondary" href="">Published Web Location</a>
-          {(this.props.loc.length > 0)
-             ? <div>
-                 {this.props.loc.map(function(url, i) {
-                   return ( <span key={i}><a href={url}>{url}</a></span> )
-                 })}
-               </div>
-             : <span>No data is associated with this publication.</span> }
+          <div className="o-textlink__secondary">Published Web Location</div>
+          <div>
+            {this.props.pub_web_loc.map(function(url, i) {
+              return ( <span key={i}><a href={url}>{url}</a></span> )
+            })}
+          </div>
         </div>
-        {/* ToDo: Hook-up Rights */}
-        <a href="" className="c-publishedlocation__license">
-          <img src="/images/icon_cc-by-nc.svg" alt=""/>
-        </a>
+    } else {
+      pub_loc_block = <div className="c-publishedlocation__location">&nbsp;</div>
+    }
+    return (
+      <div className="c-publishedlocation">
+        {pub_loc_block}
+        {this.props.rights && this.getRights(this.props.rights)}
       </div>
     )
   }
+
+  getRights = r => {
+    let baseURL = "https://creativecommons.org",
+      lv = "4.0",
```

Comment:
Oh oh, I thought this was pulling the svg file from creativecommons.org. I see now that it's the href of the <a> that's pointing to out, which makes perfect sense.